packages/feedback-spa/src/app/modules/feedback/home/feedback-home.component.ts

```diff
@@ -13,6 +13,50 @@ export class FeedbackHomeComponent implements OnInit {
   allFeedback: any;
   feedbackDetails: any;
   gitlabURL: string;
+  modules = [];
+  activeTab = null;
+  stats = [];
+  bugsCount;
+  feedbackCount;
+  totalFeedbackCount;
+  orderBy: string;
+  searchText: string;
+  feedbackPage: Number;
+  feedbackType: String = null;
+  isAccordionOpen: boolean | any = false;
+  selectedIndex: Number = 0;
+  selectedTab: Number = 0;
+  feedbackTags = [];
+  options: any = [
+    {
+      id: 0,
+      name: 'Mod Handovers'
+    },
+    {
+      id: 1,
+      name: 'Outages'
+    },
+    {
+      id: 2,
+      name: 'DSAL'
+    },
+    {
+      id: 3,
+      name: 'Feedback'
+    },
+    {
+      id: 4,
+      name: 'Projects & Initiatives'
+    },
+    {
+      id: 5,
+      name: 'Service Status'
+    }
```

Comment:
Why do we have these options here? These modules are not in the Platform, and also are not a part of the MVP.
Make these dynamic, and use a mock API call to fetch these.

Reply:
As discussed these option will come dynamically for now i have created mock file.